README.md

```
@@ -708,19 +708,27 @@ following
-    1. click the category "Meeting" and check the scope `meeting:write:meeting`.
-    2. click the category "User" and check the scope `user:read:settings`.
-11. Click "Done".
-12. You're good to go. Now you can easily add your Zoom integration in the Cal.diy settings.
+2. On the upper right, click **Develop** → **Build App**.
+3. Select **General App** and click **Create**.
+4. Name your app.
+5. Under **Select how the app is managed**, choose **User-managed app**.
+6. Click **Create** to proceed.
+7. Use the **Development** tab credentials for self-hosted Cal.diy. Copy **Client ID** and **Client Secret** into your `.env` file:
+
+   ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)